pkg/controller/node_group_test.go

```diff
@@ -471,6 +476,31 @@ func TestValidateNodeGroup(t *testing.T) {
 					HardDeleteGracePeriod:              "1h10m",
 					ScaleUpCoolDownPeriod:              "55m",
 					TaintEffect:                        "",
+					MaxNodeAge:                         "",
```

Comment:
Piggy backing this particular edge case onto the case where there's an empty TaintEffect means that us also checking that we're parsing an empty MaxNodeAge is kinda lost. Either make a new table test case here or add that we're checking this field in the name of the table test case.

Reply:
Have added a test case for this